vpr/src/base/vpr_types.h

```diff
@@ -618,19 +618,19 @@ struct t_bb {
  */
 struct t_2D_bb {
     t_2D_bb() = default;
-    t_2D_bb(int xmin_, int xmax_, int ymin_, int ymax_, int layer_num_)
+    t_2D_bb(int xmin_, int ymin_, int xmax_, int ymax_, int layer_num_)
```

Comment:
I changed the order to be consistent with vtr::Rect
I can replace the first four arguments with a vtr::Rect argument

Reply:
I reverted this change because `t_bb` uses uses the old arguement order, I think if `t_bb` and `t_2D_bb` constructors use different argument orders, it would more confusing than the mismatch between `vtr::Rect` and `t_bb`.